tests/res/enkf/test_row_scaling_case.py

```diff
@@ -297,3 +298,75 @@ def test_update_code2(self):
                     update_node1.asField(),
                     update_node2.asField(),
                 )
+
+    # This test is identical to test_update_code2(), but the row scaling is
+    # applied with the function row_scaling.scaling_assign_vector() instead of
```

Comment:
```suggestion
    # applied with the function row_scaling.assign_vector() instead of
```
This is the actual function name, right?

Reply:
Yes - that is right; I have updated.